src/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  color: var(--primary-text-color);
+
+  --accent-color: #00acdc;
+  --primary-text-color: #060b35;
+  --secondary-text-color: #616070;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: auto;
+  margin-bottom: 0 auto 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 0 4px;
```

Comment:
use `margin-bottom`

Reply:
@alexandra-protyanova 
I don't agry with you, we haven't file reset css, this margin reset basic style for tag <h2>

Reply:
okay, but your margins are incorrect anyway, image should has margin-bottom to pass the tests

Reply:
And better to create `reset.css` file to make your code clearer

Reply:
Yes, i will do it. But some mentor said me, do not create file reset.css, reset style for you tag when you need it, so i thought that i can do it in such way...

Reply:
It is better to create `reset.css`, but it is not critical for this task. Please fix your tests